docs/griptape-framework/structures/tasks.md

```diff
@@ -146,3 +146,50 @@ agent.run()
                              Output: The summary of the webpage https://www.griptape.ai has been successfully stored in a file named
                              griptape.txt.
 ```
+
+
+## Extraction Task
```

Comment:
`ExtractionTask` can be used with `CsvExtractionEngine` and `JsonExtractionEngine`. We should include examples for both.